### ISSUE TITLE
Cn you document how to deploy a React.js project to an AWS sub account

### ISSUE THREAD

**Issue by donnrri** [aws-amplify/docs#2377]
This is a great project . But I cannot find information in the Documentation about how to use the Amplify CLI to publish my React.js application to a sub-account in AWS.  I can clearly see from the Docs how to publish to the main AWS account , but I cannot see how to specify a sub-account

I could deploy my React.js application from the sub account but would loose the advantages of provisioning auth etc using the cli.

Any comments would be very welcome. Thanks

**Comment by mauerbac** [comment]
hi @donnrri - what do you mean by a subaccount ? Are you an IAM user on an account? You can pick a region and deploy using a keypair

**Comment by donnrri** [comment]
I apologise I was not clear. My company have an AWS account and have created a sub account within it.  When I set up amplify cli and tried to publish my application it deployed to the main account not within the sub account. However, all I had to do was 'switch' into the sub-account, uninstall "aws-amplify and configure it again while logged into the sub-account and then the deploy / publish go to the sub account.  Thanks for your time.
